vispy/visuals/filters/base_filter.py

```diff
@@ -120,3 +125,98 @@ def _detach(self, visual):
 
         self._attached = False
         self._visual = None
+
+
+class PrimitivePickingFilter(Filter, metaclass=ABCMeta):
```

Comment:
Any reason to use `ABCMeta` other than just inherit from `ABC`? Not really important, just curious.

Reply:
I think I chose to specify the metaclass directly because it was already inheriting from another class so this felt more explicit. I was a little back-and-forth on this because it does the exact same thing and even seeing the word "metaclass" can be a bit ominous, haha.